packages/grafana-ui/src/components/Table/Table.mdx

```diff
@@ -7,11 +7,11 @@ Used for displaying tabular data
 
 ## Sub-tables
 
-Sub-tables are supported through the usage of the prop `subData` Dataframe array.
-The frames are linked to each row using the following custom properties under `dataframe.meta.custom`
+Sub-tables are supported by adding `FieldType.nestedFrames` to the field that contains the nested data in your dataframe.
 
-- **parentRowIndex**: number - The index of the parent row in the main dataframe (under the `data` prop of the Table component)
```

Comment:
Are these being removed/deprecated?

Reply:
Good catch. `parentRowIndex ` was added for the Trace results table and I'm not sure if it's used by anyone else. Can't be sure however. I wonder is it sufficient to add a breaking change notice to the PR & change log for this or should we add a deprecation notice and migrate? @ifrost

Reply:
This is a `grafana-ui` component. Please check with the code owners (@grafana/grafana-bi-squad) on what's the deprecation policy. Might be worth checking with how we deprecate APIs in other components in `grafana-ui` with @grafana/grafana-frontend-platform.

Reply:
I added migration logic in https://github.com/grafana/grafana/pull/71953/commits/6ecbf9de3497e03da5deaa769ceee56e4976713b. The logic exists in the TablePanels adjacent `migrations.ts` file. The migration logic is used in `TableContainer.tsx` & `TablePanel.tsx`. Basically if the supplied data frames contain `meta.custom.parentRowIndex` they will be converted to the new nested format with `FieldType.nestedFrames` instead.

I also added a deprecation notice to `Table.mdx`. Not sure if this is the correct location or is enough to express a deprecation? It does not mention when the removal of the old `parentRowIndex` will take place.

Hopefully someone from the Bi Squad / Frontend Platform can offer help/guidance on how to deprecate something like this.

Reply:
I think this and a change log note should cover it.

Reply:
AFAIK, the change log is auto generated if you add the `Add to changelog` label to the PR - so in that case it just uses the PRs title.